.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Ballerina Build
+        uses: ballerina-platform/ballerina-action/@master
+        with:
+          args:
+            build -a -c --skip-tests
+        env:
+          JAVA_OPTS: -DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true
```

Comment:
```suggestion

```
This line is only added to connectors with organization, _ballerina_

Reply:
Fixed.